src/Singular.jl

```diff
@@ -90,6 +90,14 @@ function __init__()
    # Initialise Singular
    ENV["SINGULAR_EXECUTABLE"] = binSingular
    libSingular.siInit(binSingular)
+
+   # don't tell the user about each library that singular loads
+   libSingular.set_option("V_LOAD_LIB", false)
+   # don't tell the user when a library redefines a singular variable
+   libSingular.set_option("V_REDEFINE", false)
+   # silence printlevel-based printing
+   libSingular.set_printlevel(-100)
```

Comment:
Setting the printlevel should not be necessary: it starts with 0, gets decreased with each procedure call,
and should print something only for positive values.

Reply:
I am looking at this line
https://github.com/Singular/Singular/blob/spielwiese/Singular/LIB/normal.lib#L161
which needs to be negative for any (positive) value of voice.
I have seen other libraries that have a +3

Reply:
What is the status here?

Reply:
If the libraries do not follow the conventions....we can use this work-around.
SO setting the printlevel to -100 is fine with me.

Reply:
The libsingular_julia has my options PR in it already, so if you would like to have this Singular.jl PR, we would need a new version of libsingular_julia and then a new version of Singular.jl. Right?

Reply:
Yes. Also, we should set things up so that the ideas originally by @benlorenz and @thofma (to test "branches against branches with same name" from the OSCAR workshop are used for both repositories, in combination with https://github.com/oscar-system/libsingular-julia/pull/17 so that we can test the combined changes to libsingular_julia and Singular.jl together, and *before* making a libsingular_julia release.

If someone were to help with implementing that extra testing that'd be great